scripts/sources/parameters/cmd/aws_ssm.go

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Why so much nesting: `scripts/sources/parameters/cmd/aws_ssm.go`. Can you just put in `cmd/aws_ssm/main.go`?